nixos/modules/services/x11/desktop-managers/kde5.nix

```diff
@@ -229,6 +229,8 @@ in
       # Enable helpful DBus services.
       services.udisks2.enable = true;
       services.upower.enable = config.powerManagement.enable;
+      services.dbus.packages =
```

Comment:
@bjornfor The D-Bus service should be installed automatically already; have you checked if this is the case?